### ISSUE TITLE
Skip Forward and Back from iOS Lock Screen Not Working

### ISSUE THREAD

**Issue by MerryOscar** [luanpotter/audioplayers#801]
Skipping forward and back 30 seconds from the iOS lock screen does not work properly.

**Full Description:**
Skipping forward and back 30 seconds using the in-app controls works well through the seek method:
```dart
_player.seek(_position.inSeconds + 30)
```
However after setting the iOS lock screen controls via:
```dart
_player.setNotification(
  title: 'Test Audio',
  forwardSkipInterval: Duration(seconds: 30),
  backwardSkipInterval: Duration(seconds: 30),
  duration: Duration(seconds: duration),
  elapsedTime: Duration.zero,
);
```
The lock screen controls move the position forward and backward by **30 milliseconds instead of 30 seconds**.

**Steps to Reproduce**:
1. Run the example project below.
2. Play the test audio file.
3. Observe the in-app skip controls working at **30 second intervals**.
4. Pause the audio and rewind to 0 seconds.
5. Use the lock screen controls to try and move forward by 30 seconds and observe that the position instead moves forward by **30 milliseconds**.

**Code to Reproduce:**
https://github.com/MerryOscar/testaudioplayers

**Platforms:**
* OS: **iOS**
* OS version: **iOS 14.4**
* Device: **iPhone Xr**
* flutter version: **1.22.4**
* audioplayers version: **0.17.4**
* release: **issue occurs both in debug and release**

**Comment by luanpotter** [comment]
Ah, good catch! Seems it would be a very simple fix? a missing factor of 1000 somewhere?

**Comment by MerryOscar** [comment]
Aah yes I think I found it!

Both these lines pass the `interval` parameter as `millis` rather than `seconds`:

https://github.com/luanpotter/audioplayers/blob/659e467fc3db07f2414a9cb91bc1432bb681e67c/darwin/Classes/WrappedMediaPlayer.swift#L143

https://github.com/luanpotter/audioplayers/blob/659e467fc3db07f2414a9cb91bc1432bb681e67c/darwin/Classes/WrappedMediaPlayer.swift#L156

**Comment by MerryOscar** [comment]
@luanpotter could you take a look at #804 - I've tested and it fixes the issue.

**Comment by MerryOscar** [comment]
#804 caused an error. #812 is working correctly.

**Comment by erickzanardo** [comment]
Fixed on #804
